.travis.yml

```diff
@@ -32,9 +32,3 @@ deploy:
     - npm run publish-semantic-release
   on:
     all_branches: true
-
-branches:
-  only:
-    - main
-    - next
-    - /^v.*$/
```

Comment:
Just so I'm clear, this means that semantic release can publish on any branch now? Do we run the risk of releasing a `renovate` branch this way? (probably not since we always merge those into other branches)

Reply:
Yes, this is potentially true. The checks in place for semantic releases are:

1. does the branch exist in the `branches` config? If not, Travis doesn't run for that branch
2. does the branch exist in the `deploy` config in Travis? If not, the deploy step doesn't run for that branch
3. does the branch exist in the Semantic Release config? If not, semantic release will skip that branch

At this stage (3) is holding us down, so that Renovate won't release a version on its branch (and even if it did, it would have a special dist-tag, i.e. not `latest`)

Before we had (1) and (3) as safety checks, and now we only have (3). I could update this to add a check for (2) back if you think that's a good idea?

Reply:
Ah, I forgot about the `branches` field in the Semantic Release config. I think (3) alone is sufficient in this case 👍